lnrpc/watchtowerrpc/watchtower.proto

```diff
@@ -1,18 +1,24 @@
 syntax = "proto3";
 
+import "google/api/annotations.proto";
+
 package watchtowerrpc;
 
 option go_package = "github.com/lightningnetwork/lnd/lnrpc/watchtowerrpc";
 
 // Watchtower is a service that grants access to the watchtower server
 // functionality of the daemon.
 service Watchtower {
-    /* lncli: tower info
+    /* lncli: `tower info`
     GetInfo returns general information concerning the companion watchtower
     including its public key and URIs where the server is currently
     listening for clients.
     */
-    rpc GetInfo (GetInfoRequest) returns (GetInfoResponse);
+    rpc GetInfo (GetInfoRequest) returns (GetInfoResponse) {
+        option (google.api.http) = {
+            get: "/v2/watchtower"
```

Comment:
Perhaps add a `/server` prefix here to disambiguate from the client calls below?